src/Application/UI/PresenterComponentReflection.php

```diff
@@ -125,7 +125,7 @@ public static function combineArgs(\ReflectionFunctionAbstract $method, $args)
 			} else {
 				$res[$i++] = $arg = isset($args[$name]) ? $args[$name] : NULL;
```

Comment:
If the items in `$method->getParameters()` are indexed by numbers starting from zero then yes, it would be better.